pages/agent/v3/securing.md.erb

```diff
@@ -133,3 +133,16 @@ Environment=https_proxy=http://username:password@proxyserver:8080/
 {: codeblock-file="proxy.conf"}
 
 After creating this file, systemd will require a reload and the `buildkite-agent` service will require a restart. 
+
+## Preventing Buildkite infrastructure compromise
+ 
+Suppose your Buildkite agent is being deployed into a sensitive environment. In that case, there are a few default settings that you'd want to adjust to specifically safeguard the rest of your Buildkite Organization's infrastructure in case of the agent's compromise (for example, if the agent API gets compromised).
+To keep your Buildkite Organization's infrastructure safe in case of Buildkite agent's compromise:  
+
+* Disallow execution of arbitrary commands by [customizing the list of allowed plugins](/docs/agent/v3/securing#allowing-a-list-of-plugins) or outright [disable plugins](/docs/agent/v3/securing#disabling-plugins)  
+* [Disable command-eval](/docs/agent/v3/securing#disabling-command-eval)  
+* [Disable local hooks](/docs/agent/v3/securing#disabling-local-hooks)  
+* [Enable strict validation in agent hooks](/docs/agent/v3/securing#strict-checks-using-agent-hooks)  
+ 
+As a result, your Buildkite agent will refuse to run anything that's not a single argumentless invocation of a script that exists locally (after the `git clone` step of the setup) unless it's explicitly allowed by you. 
+Since the [agent](https://github.com/buildkite/agent) is open-source, if necessary, you can verify that assertion to whatever degree of certainty is required.  
```

Comment:
```suggestion
Since the [agent](https://github.com/buildkite/agent) is open-source, if necessary you can verify that assertion to whatever degree of certainty is required.  
```

Reply:
So, it looks like this one can be merged now, too?